src/App.jsx

```diff
@@ -3,7 +3,7 @@ import { Route, Switch, Redirect } from "react-router-dom";
 import { withRouter } from "react-router";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import NotFound from "./pages/404/NotFound";
+import NotFound from "./pages/404/notFound";
```

Comment:
its is 

`import NotFound from "./pages/404/NotFound";`

Not

`import NotFound from "./pages/404/notFound";`

Reply:
in this repo there is NotFound.jsx (NOT notFound.jsx)

see the screenshot
![image](https://user-images.githubusercontent.com/60546840/115279777-fbff9a80-a0fb-11eb-9145-a9911a6a5cfd.png)

Reply:
there is no notFound.jsx

I m still confused.. how come deploy preview successfully compiled this

Reply:
I also don't know for some reason the file name after pulling from the upstream was notFound instead of NotFound so that's why I changed the name in the import statement.

Reply:
no worries. just change it back.